student.rb

```diff
@@ -1,14 +1,19 @@
 require_relative 'person'
 
 class Student < Person
-  attr_accessor :classroom
+  attr_reader :classroom
 
-  def initialize(id, age, classroom, name = 'Unknow')
+  def initialize(id, age, name = 'Unknow')
```

Comment:
- [ ] [OPTOINAL] The value of `name` should be equal to `Unknown` you wrote `Unknow` please fix that type error